Makefile

```diff
@@ -5,13 +5,13 @@ dist/rokka-cli.phar: vendor box.json tools/box bin/rokka-cli $(SRC_FILES) compos
 	./tools/box compile --quiet
 
 tools/box:
-	wget --directory-prefix=tools --quiet https://github.com/box-project/box/releases/download/3.14.0/box.phar
+	wget --directory-prefix=tools --quiet https://github.com/box-project/box/releases/download/3.14.1/box.phar
```

Comment:
that is the latest release. i tried with both locally but always get an error `✘ The application requires the version "^7.2" or greater.` - not sure why, as we clearly require php 8, and box itself afaik should run on php 8 too.